api/src/main/java/uk/gov/dhsc/htbhf/claimant/entitlement/PregnancyEntitlementCalculator.java

```diff
@@ -29,4 +33,23 @@ public boolean isEntitledToVoucher(LocalDate dueDate, LocalDate entitlementDate)
         LocalDate endOfGracePeriod = dueDate.plusWeeks(pregnancyGracePeriodInWeeks);
         return !endOfGracePeriod.isBefore(entitlementDate);
     }
+
+    /**
+     * Determines if the current cycle is the claimant's second to last cycle with pregnancy vouchers.
+     *
+     * @param currentPaymentCycle the current payment cycle
+     * @return true if the current and next payment cycle are entitled to pregnancy vouchers, and the one after that is not entitled to vouchers.
+     */
+    public boolean currentCycleIsClaimantsSecondToLastCycleWithPregnancyVouchers(PaymentCycle currentPaymentCycle) {
```

Comment:
We may not need the word `Claimants` in the method name.

Reply:
Removed claimants.